modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/segregated/2d/tests

```diff
@@ -0,0 +1,97 @@
+[Tests]
+  issues = '#22356'
+  design = 'SIMPLE.md'
+  [momentum]
+    type = 'Exodiff'
+    input = 2d-segregated-velocity.i
+    exodiff = 2d-segregated-velocity_out.e
+    requirement = "The system shall be able to solve the steady-state Navier-Stokes problem in a 2D "
+                  "channel using the SIMPLE algorithm with separating the momentum components into "
+                  "different systems."
+    recover = false # we don't support recovery for SIMPLE yet
+  []
+  [rz]
+    type = 'Exodiff'
+    input = 2d-segregated-velocity-rz.i
+    exodiff = 2d-segregated-velocity-rz_out.e
+    requirement = "The system shall be able to solve the steady-state Navier-Stokes problem in a 2D "
+                  "cylindrical channel using the SIMPLE algorithm."
+    abs_zero = 1e-9
+    recover = false # we don't support recovery for SIMPLE yet
+  []
+  [slip]
+    type = 'Exodiff'
+    input = 2d-segregated-velocity-rz-slip.i
+    exodiff = 2d-segregated-velocity-rz-slip_out.e
+    requirement = "The system shall be able to solve the steady-state Navier-Stokes problem in a 2D "
+                  "cylindrical channel with slip boundary conditions using the SIMPLE algorithm."
+    recover = false # we don't support recovery for SIMPLE yet
+  []
+  [slip-with-friction]
+    type = 'Exodiff'
+    input = 2d-segregated-velocity-rz-slip.i
+    exodiff = 2d-segregated-velocity-rz-slip-friction.e
+    cli_args = "FVKernels/inactive='' Outputs/file_base='2d-segregated-velocity-rz-slip-friction'"
+    requirement = "The system shall be able to solve the steady-state Navier-Stokes problem in a 2D "
+                  "cylindrical channel with homogenized friction using the SIMPLE algorithm."
+    recover = false # we don't support recovery for SIMPLE yet
+  []
+  [with-energy]
+    type = 'Exodiff'
+    input = 2d-segregated-energy.i
+    exodiff = 2d-segregated-energy_out.e
+    requirement = "The system shall be able to solve the steady-state Navier-Stokes problem together "
+                  "with the energy equation in a 2D channel using the SIMPLE algorithm."
+    recover = false # we don't support recovery for SIMPLE yet
+  []
+  [with-energy-symmetry]
+    type = 'Exodiff'
+    input = 2d-segregated-energy.i
+    exodiff = 2d-segregated-energy-symmetry.e
+    # We need to force the boundary execution on the bottom surface because the segregated RC will have a
+    # small residual flux crossing the boundary
```

Comment:
so what's up with that?

Reply:
It is what it is, not ideal for sure. I checked and onter CFD solvers have it too. I will keep think on corrections to get rid of this.